install_config/install/disconnected_install.adoc

```diff
@@ -0,0 +1,550 @@
+= Disconnected Installation
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Frequently, portions of a datacenter may not have access to the Internet, even via proxy servers. Installing OpenShift in these environments is considered a disconnected installation.
+
+An OpenShift Enterprise disconnected installation differs from a regular
+installation two primary ways:
+
+* The OpenShift software channels and repositories are not available via Red Hat’s content distribution network. 
+* OpenShift uses several Dockerized components. Normally, these images are pulled directly from Red Hat’s Docker registry. In a disconnected environment, this is not possible.
+
+A disconnected installation ensures the OpenShift software is made available to the relevant servers, then follows the same installation process as a standard connected installation. This topic additionally details manually downloading the Docker images and how to transport them onto the relevant servers. 
```

Comment:
For the last sentence, suggest: "This topic additionally details how to manually download the Docker images and transport them onto..."